WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsStripeAccountOverdueView.swift

```diff
@@ -13,9 +15,28 @@ struct InPersonPaymentsStripeAccountOverdue: View {
             ),
             supportLink: true,
             learnMore: true,
-            analyticReason: analyticReason
+            analyticReason: analyticReason,
+            buttonViewModel: InPersonPaymentsOnboardingErrorButtonViewModel(text: Localization.primaryButtonTitle,
+                                                                            analyticReason: analyticReason,
+                                                                            action: {
+                                                                                presentedSetupURL = setupURL
+                                                                            }),
+            secondaryButtonViewModel: InPersonPaymentsOnboardingErrorButtonViewModel(text: Localization.secondaryButtonTitle,
+                                                                                     analyticReason: analyticReason,
+                                                                                     action: {
+                                                                                         onRefresh()
+                                                                                     })
         )
+        .safariSheet(url: $presentedSetupURL, onDismiss: onRefresh)
      }
+
+    private var setupURL: URL? {
```

Comment:
I think this belong to the view model

Reply:
Agree, at the moment seems we're not using a view model in any of the dozen+ views for onboarding state, we only expose a couple of properties, and the callback closure. I'll log this separately because we can do the same changes for multiple views both for `setupURL` and others: https://github.com/woocommerce/woocommerce-ios/issues/10679